pkg/networkmanager/firewall.jsx

```diff
@@ -333,7 +335,12 @@ export class Firewall extends React.Component {
             );
         }
 
-        var services = [...this.state.firewall.enabledServices].map(id => this.state.firewall.services[id]);
+        var services = [...this.state.firewall.enabledServices].map(id => {
+            var service = this.state.firewall.services[id];
```

Comment:
Same here. Also, please don't use `var` in arrow functions, the scoping is completely broken. Use `let` or `const`.